appeals/web/src/server/lib/mappers/appeal/submappers/lpa-final-comment-due-date.mapper.js

```diff
@@ -11,8 +11,8 @@ export const mapLpaFinalCommentDueDate = ({
 		id: 'lpa-final-comment-due-date',
 		text: 'LPA final comments due',
 		// @ts-ignore
```

Comment:
The same type fix as mentioned elsewhere applies here (just need to add `lpaFinalCommentDueDate` to `WebAppealTimetable`)